hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleCreateSpecs.java

```diff
@@ -203,12 +208,13 @@ private HapiSpec onlyBodyAndMemoCreation() {
                         .hasRecordedScheduledTxn());
     }
 
+    @HapiTest
     private HapiSpec bodyAndPayerCreation() {
         return defaultHapiSpec("BodyAndPayerCreation")
                 .given(cryptoCreate(PAYER))
                 .when(scheduleCreate(
                                 ONLY_BODY_AND_PAYER,
-                                cryptoTransfer(tinyBarsFromTo(DEFAULT_PAYER, GENESIS, 1))
+                                cryptoTransfer(tinyBarsFromTo(PAYER, GENESIS, 1))
```

Comment:
Why is PAYER insteadof DEFAULT_PAYER here? Didn't check the code in detail. Are those two different values?

Reply:
Yes, different values, and default payer doesn't match the created payer on line 214.  It *sometimes* worked due to ordering assumptions; this makes it correctly work regardless of execution order.

Reply:
Not sure why DEFAULT_PAYER was used here before. Was this test passed in mono code run?

Reply:
It passed in mono (and still does); not entirely clear why (possibly due to the prior tests setting the DEFAULT_PAYER into just the right state), but it failed in Hapi and the mismatch was clearly unintentional (possibly an ancient copy/paste that just happened to succeed?).
It's surprising how much of the E2E and Integration tests pass almost due to luck rather than intent.